v1.0/v1.0/envvar3.cwl

```diff
@@ -6,6 +6,9 @@ outputs:
   results:
     type: File
     outputBinding: { glob: results }
+  # log:
+  #   type: File
+  #   outputBinding: { glob: log }
```

Comment:
Better to remove it than comment it out?

Reply:
It is nice when one is debugging this particular test.